### PR TITLE
plat-ls:add LS1028A-RDB platform

### PR DESCRIPTION
**Pull request by sahilnxp**

### CODE REVIEW

File: .shippable.yml [hunk 1]
```diff
@@ -95,6 +95,7 @@ build:
     - _make PLATFORM=ls-ls1088ardb
     - _make PLATFORM=ls-ls2088ardb
     - _make PLATFORM=ls-lx2160ardb
+    - _make PLATFORM=ls-ls1028ardb
```
Comment (jbech-linaro) [inline .shippable.yml]: Can you just move it up before the `ls-ls1088ardb`, so we keep them somewhat sorted in alphabetic order?

File: MAINTAINERS [hunk 2]
```diff
@@ -92,8 +92,9 @@ MediaTek MT8173 EVB
 S:	Orphan
 F:	core/arch/arm/plat-mediatek/
 
-NXP LS1021A, LS1043A-RDB, LS1046A-RDB, LS1012A-RDB, LS1012A-FRWY, LS1088A-RDB, LS2088A-RDB, LX2160A-RDB
+NXP LS1021A, LS1043A-RDB, LS1046A-RDB, LS1012A-RDB, LS1012A-FRWY, LS1088A-RDB, LS2088A-RDB, LX2160A-RDB, LS1028A-RDB
```
Comment (jbech-linaro) [inline MAINTAINERS]: Same here, about sorting it.

**Comment by jbech-linaro** [comment]
Just two minor comments, otherwise, please also file a patch that updates the [docs](https://optee.readthedocs.io/general/platforms.html).
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by sahilnxp** [comment]
Comments taken care and Tag applied.
Raising a pull request to update docs in a while.

**Comment by sahilnxp** [comment]
https://github.com/OP-TEE/optee_docs/pull/30 For updating docs.

**Comment by jforissier** [comment]
Missing space between `plat-ls:` and `add LS1028ARDB platform`.

Minor: you may also fix the checkpatch warning (Travis CI error) by writing a small blurb in the descriptions, such as "Adds support for the The Layerscape® LS1028A reference design board (LS1028ARDB) from NXP."

**Comment by sahilnxp** [comment]
PR updated

**Comment by jforissier** [comment]
Thanks @sahilnxp!

**Pull request merged**